interface/main/calendar/add_edit_event.php

```diff
@@ -42,7 +43,7 @@
 require_once($GLOBALS['srcdir'].'/encounter_events.inc.php');
 require_once($GLOBALS['srcdir'].'/acl.inc');
 require_once($GLOBALS['srcdir'].'/patient_tracker.inc.php');
-
+require_once($GLOBALS['incdir']."/main/holidays/Holidays_Controller.php");
```

Comment:
Recommend only using lowercase in script file names (unless using classes that require this).